src/libraries/System.IO.MemoryMappedFiles/src/Microsoft/Win32/SafeMemoryMappedFileHandle.Unix.cs

```diff
@@ -76,22 +67,19 @@ public sealed partial class SafeMemoryMappedFileHandle : SafeHandleZeroOrMinusOn
             SetHandle(handlePtr);
         }
 
-        protected override void Dispose(bool disposing)
```

Comment:
Is this going be an observable behavior change for user defined FileStreams? The potential user-defined Dispose is not going to be called anymore.

Reply:
Yes, although that looks like it's already a discrepancy between our Windows and Unix implementation, with the Windows implementation not Dispose'ing of the file stream and the Unix implementation doing so (unless I'm missing it in the code).  While it would then be a behavior change on Unix, as the Unix implementation is trying to match the pre-existing Windows semantics, seems like a reasonable bug fix.